src/agent/rustjail/src/mount.rs

```diff
@@ -1224,4 +1267,86 @@ mod tests {
 
         assert!(check_proc_mount(&mount).is_err());
     }
+    #[test]
+    fn test_rootfs_not_exists() {
+        let rootfs = "/home/rootfs";
+        assert_eq!("/home/rootfs/a/b/c", secure_join(rootfs, "a/b/c"));
+    }
+
+    #[test]
+    fn test_rootfs_relative() {
+        let rootfs = "/home/rootfs";
+        assert_eq!("/home/rootfs/a/b/c", secure_join(rootfs, "../../../a/b/c"));
+    }
+
+    #[test]
+    fn test_rootfs_strip_any_relative() {
+        let rootfs = "/home/rootfs";
+        assert_eq!(
+            "/home/rootfs/a/b/c",
+            secure_join(rootfs, "../../../a/../../b/../../c")
+        );
+    }
+
+    #[test]
+    fn test_rootfs_relative_softlink_1() {
+        let rootfs = "/tmp/rootfs1";
+        // /var/run/1 -> ../../../../
+        let unsafe_path = "/var/run/1";
+
+        // prepare softlink to test
+        remove_dir_all("/tmp/rootfs1").unwrap_or_default();
+        create_dir_all("/tmp/rootfs1/var/run").expect("failed to create rootfs");
+        fs::symlink("../../../../", "/tmp/rootfs1/var/run/1").unwrap();
```

Comment:
These tests don't clean up - the symlink need to be removed on success. Again, table-based tests make this a one-liner with minimal duplicated code.

Reply:
Ok. Will be updated to table-based tests

Reply:
Done